update_duties/update_duties.py

```diff
@@ -0,0 +1,53 @@
+"""
+Module containing functions to update duty table in database according to current state of the API
+data.
+"""
+
+from functools import partial
+
+from dotenv import load_dotenv
+import pandas as pd
+import requests
+
+import database_functions as dbf
+
+
+BASE_URL = "https://data-eng-plants-api.herokuapp.com/plants/"
+
+
+def get_api_botanist_name_by_plant_id(plant_id: int) -> list[dict]:
+    """Gets plant data from API using ID."""
+    api_botanist_info = requests.get(BASE_URL + str(plant_id),
```

Comment:
small thing - if you import 'get' from requests then you can just write get() when you call the API

Reply:
good point